src/wgpu/wgpu-shader-program.cpp

```
@@ -42,10 +42,10 @@ Result ShaderProgramImpl::createShaderModule(slang::EntryPointReflection* entryP
     wgslDesc.chain.sType = WGPUSType_ShaderSourceWGSL;
     wgslDesc.code = WGPUStringView{module.code.c_str(), module.code.size()};
 #endif
-    WGPUShaderModuleDescriptor desc = {};
-    desc.nextInChain = (WGPUChainedStruct*)&wgslDesc;
+    WGPUShaderModuleDescriptor moduleDesc = {};
+    moduleDesc.nextInChain = (WGPUChainedStruct*)&wgslDesc;
 
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)